notebooks/official/ml_metadata/vertex-pipelines-ml-metadata.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<p><strong>AI Platform Notebooks -&gt; Vertex AI Workbench Notebook</strong></p><p> </p><p>Combine the two text cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/826/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>